packages/components/src/Checkbox/Checkbox.css

```diff
@@ -1,12 +1,11 @@
-.checkbox {
-  --checkbox-size: var(--space-base);
+.checkBoxParent {
+  --checkbox--size: 20px;
+  --checkbox--border--checked: var(--color-interactive);
```

Comment:
Setting these as variables gets rid of a ton of messy selector-nesting to manage state

Reply:
It doesn't necessarily show up in the diff because I didn't commit alternatives, but a lot of similar-looking states have varying levels of specificity, so I had found myself either declaring the same value multiple times for different selectors throughout the file, or disable linting and over-riding specificity warnings.